deploy-board/deploy_board/webapp/accounts.py

```diff
@@ -5,7 +5,7 @@
 def create_legacy_ui_account(account_id):
     if account_id == AWS_PRIMARY_ACCOUNT:
         return {
-            "legacy_name": f"{AWS_PRIMARY_ACCOUNT} / Primary AWS account",
```

Comment:
I think we should do the other way around, ie. keep Primary AWS account and make all the `/ default` one to be ` Primary AWS account`

Reply:
sounds good, I'll address this in rodimus then since that's where the default value comes from.